client/src/components/sideBar/SideBar.js

```diff
@@ -153,7 +153,7 @@ export function SideBar() {
     return (
       <span
         className={`fixed top-0 z-50 flex h-full items-center transition-all ${
-          isSidebarOpen ? 'left-[20vw]' : 'left-0'
+          isSidebarOpen ? 'left-[260px]' : 'left-0'
```

Comment:
this is fine but just curious why we're hardcoding the width? was the sidebar too small on small screens?

Reply:
check out my reasonings here #99 
open to discussion